review/ha_basic.toml

```diff
@@ -0,0 +1,44 @@
+[server]
+hostname = "wso2is.example.com"
+proxy_port = "443"
+local_ip = "192.168.1.1"
+
+[super_admin]
+username = "admin"
+password = "admin"
+
+[user_store]
+type = "read_write_ldap"
+connection_url = "ldap://ldap.example.com:389"
+connection_name = "uid=admin,ou=wso2is"
+connection_password = "$secret{ldap_password}"
+base_dn = "dc=example,dc=com"
+
+[database.core_db]
+type = "mysql"
+hostname = "mysql.example.com"
+database = "wso2regdb"
+password = "$secret{db_password}"
+
+[database.identity_db]
+type = "mysql"
+hostname = "mysql.example.com"
+database = "wso2isdb"
+password = "$secret{db_password}"
+
+[keystore.tls]
+name = "keystore.jks"
+password = "$secret{keystore_password}"
+
+[keystore.crypto]
+name = "keystore.jks"
+password = "$secret{key_password}"
+
+[cluster]
+scheme = "wka"
+members = ["192.168.1.1:4000","192.168.1.2:4000"]
+
+[secrets]
+db_password = "[R!#etoqe]"
```

Comment:
still do not see why we need this section, should not we refer this from the secure_vault

Reply:
These values are added to the secure vault upon encryption and will be used from there in the same way as done now.
The reason we thought of having this here is to provide the user the ease of configuration by having all in a single file.